README.md

```diff
@@ -1,11 +1,11 @@
 
-ECMAScript2015(ES6) is the latest version of the ECMAScript standard. 
+ECMAScript 2016 (ES7) is the latest version of the ECMAScript standard. 
```

Comment:
No, ES7 is the latest version (only two features added). We could say something like "ES6 and beyond is popular"